common-utils/src/main/java/com/common/ledger/utils/LedgerResponse.java

```diff
@@ -0,0 +1,134 @@
+/**
+ * 
+ */
+package com.common.ledger.utils;
+
+import static com.common.ledger.utils.ResponseUtil.RESPONSEMAP_CODE_KEY;
+import static com.common.ledger.utils.ResponseUtil.RESPONSEMAP_DATA_KEY;
+import static com.common.ledger.utils.ResponseUtil.RESPONSEMAP_ERRORCODE_KEY;
+import static com.common.ledger.utils.ResponseUtil.RESPONSEMAP_MESSAGE_KEY;
+import static com.common.ledger.utils.ResponseUtil.RESPONSEMAP_PATH_KEY;
+import static com.common.ledger.utils.ResponseUtil.RESPONSEMAP_STATUS_KEY;
+import static com.common.ledger.utils.ResponseUtil.RESPONSEMAP_TIMESTAMP_KEY;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.springframework.http.HttpStatus;
+
+public class LedgerResponse implements Serializable {
```

Comment:
We can discuss on this class

Reply:
we just trying to maintain some standard response in all services so we are using this util module